### ISSUE TITLE
When stopping the engine with ctrl + c after some time passes an error appears before shutting down

### ISSUE THREAD

**Issue by ligadata-william** [LigaData/Kamanja#1044]
When I use ctrl + c to shutdown the engine, after a period of doing nothing (from the user perspective at least) the following error occurs before finishing shutdown:
```
Exception in thread "pool-6-thread-1" java.lang.NullPointerException
	at com.ligadata.KamanjaManager.PostMessageExecutionQueue$$anon$1.run(ExecContext.scala:671)
	at scala.actors.threadpool.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1060)
	at scala.actors.threadpool.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:574)
	at java.lang.Thread.run(Thread.java:745)
///////////////////////////dispose
dispose
///////////////////////////dispose
dispose
```

**Comment by ligadata-pokuri** [comment]
Fixed this in 1.4.0.Test with checkin bade97b139c13300723e9cb80a083d73e1986a6e

**Comment by ligadata-william** [comment]
Verified as fixed. Closing.
